CHANGELOG/CHANGELOG-1.0.0-rc1.md

```diff
@@ -0,0 +1,40 @@
+# Changelog since v0.2.2 (Beta.2)
+
+## Urgent Upgrade Notes
+
+### (No, really, you MUST read this before you upgrade)
+
+- `--kubelet-pod-path` and `--kubelet-csi-plugins-path` parameters for csi-proxy.exe is replaced with `--kubelet-path`. No changes are necessary if default values for the parameters were being used to initialize csi-proxy.exe
+- All CSI Plugins (and other clients of csi-proxy) using v1alpha and v1beta named pipes and APIs should continue to function as is.
+
+## Changes by Kind
+
+### Feature
+
+- Combine plugin and pod paths into one KubeletPath ([#150](https://github.com/kubernetes-csi/csi-proxy/pull/150), [@mauriciopoppe](https://github.com/mauriciopoppe))
+- Make GPT the new partition style for volumes ([#128](https://github.com/kubernetes-csi/csi-proxy/pull/128), [@manueltellez](https://github.com/manueltellez))
+- Add NewClientWithPipePath per API that can communicate with a custom named pipe ([#124](https://github.com/kubernetes-csi/csi-proxy/pull/124), [@mauriciopoppe](https://github.com/mauriciopoppe))
+- Return disk serial number in ListDiskIDs ([#116](https://github.com/kubernetes-csi/csi-proxy/pull/116), [@wongma7](https://github.com/wongma7))
+- Update klog to version 2 ([#114](https://github.com/kubernetes-csi/csi-proxy/pull/114), [@jingxu97](https://github.com/jingxu97))
+- Support reporting Disk IDs with CodeSet StorageIDCodeSetBinary ([#110](https://github.com/kubernetes-csi/csi-proxy/pull/110), [@gab-satchi](https://github.com/gab-satchi))
+
+### API
+- v1beta2 SMB API group with error field removed from all API response msgs ([#146](https://github.com/kubernetes-csi/csi-proxy/pull/146), [@mauriciopoppe](https://github.com/mauriciopoppe))
+- v1beta2 FileSystem API group with renamed APIs LinkPath -> CreateSymlink and IsMountPoint -> IsSymlink and error field removed from all API response msgs ([#143](https://github.com/kubernetes-csi/csi-proxy/pull/143), [@mauriciopoppe](https://github.com/mauriciopoppe))
+- v1beta3 Disk API group with renamed APIs DiskStats -> GetDiskStats, SetAttachState -> SetDiskState and SetAttachState -> GetDiskState ([#140](https://github.com/kubernetes-csi/csi-proxy/pull/140), [@mauriciopoppe](https://github.com/mauriciopoppe))
```

Comment:
There were a few changes in the message fields too, the one that's important was changing disk_id (string) to disk_number (uint32) in Volume and Disk, these change are summarized in the github issues attached to each PR.

I think we could add something like:

```
The complete list of changes is summarized in #[issue number]
```